src/xdocs/writingchecks.xml

```diff
@@ -115,6 +115,107 @@
 
     </section>
 
+      <section name="Tool to print Javadoc tree structure">
```

Comment:
We need to include only following things in content
command-java -cp checkstyle-7.7-SNAPSHOT-all.jar com.puppycrawl.tools.checkstyle.gui.Main MyClass.java
and following image
![image](https://cloud.githubusercontent.com/assets/6566854/23484265/4a49a70e-fefc-11e6-99b5-7772ec9eb4d6.png)
i will have to remove everything else because that is for java doc.
Am i right?